Task_HelloWorld/index.html

```diff
@@ -7,5 +6,11 @@
 
```

Comment:
The opening '<p>' tag does not have a closing '</p>' tag. It should be added for proper HTML formatting.